site/content/extend/plugins/community-plugin-marketplace.md

```diff
@@ -0,0 +1,80 @@
+---
+title: Community Plugins in the Marketplace
+subsection: Plugins (Beta)
+weight: 60
+---
+
+Once your plugin has reached a certain level of quality, you might consider submitting it to the Plugin Marketplace. The Plugin Marketplace allows discovery, installation and updates of plugins directly within Mattermost. It is a great way to get feedback on your plugin and help make it more popular. Once your plugin is accepted to the Marketplace, Mattermost will also send you swag!
```

Comment:
```suggestion
Once your plugin has reached a certain level of quality, you can consider submitting it to the Plugin Marketplace. The Plugin Marketplace is a platform that supports discovery, installation, and updates of plugins directly within Mattermost via the System Console. It's a great way to get feedback on your plugin and help make it more popular. Once your plugin is accepted to the Marketplace, Mattermost will also send you swag!
```

Question: 
- Do we have a definition of "certain level of quality"? It might be helpful to provide some guidelines around that. 
- I wonder if linking [to this page](https://docs.mattermost.com/administration/plugins.html) would help with some initial guidance around plugins? Or it might add a layer of complexity? I'm 3/5 mainly because it might help to give more info upfront.

Reply:
FYI: The Marketplace is not part of the Sytem Console and is instead a modal in the main window.

> Do we have a definition of "certain level of quality"? It might be helpful to provide some guidelines around that.

The list of requirements below defines the "certain level of quality" that is needed.

> I wonder if linking to this page would help with some initial guidance around plugins? Or it might add a layer of complexity? I'm 3/5 mainly because it might help to give more info upfront.

The page this PR adds is quite deep into the document. I don't expect non-dev to accidentally stubble across it. The page you linked is more for admins then it is for developers.